File: js/programmers/level_0/중복된 숫자 개수.js
// Lv. 0 중복된 숫자 개수

/**
 * @param {number[]} array
 * @param {number} number
 * @return {number}
 */

const solution = (array, n) => {
    return array.filter((v) => v === n).length;
};

File: java/programmers/level_0/src/중복된_숫자_개수.java
public class 중복된_숫자_개수 {

  public int solution(int[] array, int n) {
    int answer = 0;
    for (int i = 0; i < array.length; i++) {
      if (array[i] == n) {
        answer++;
      }
    }
    return answer;
  }
}
